roles/multi-tenancy/tasks/multi-tenant-onboard-offboard.yaml

```diff
@@ -146,8 +144,16 @@
   ansible.builtin.shell: |
     kubectl --kubeconfig {{ KUBECONFIG }} get pods -n {{ NAMESPACE }} --sort-by=.metadata.creationTimestamp | tac | grep sas-tenant-offboard | awk '{print $3}'
   register: pod_status
-  until: pod_status.stdout_lines[0] == "Completed"
-  retries: "{{ tenant_count|int }}"
-  delay: 75
+  until: pod_status.stdout_lines[0] == "Completed" or pod_status.stdout_lines[0] not in pod_fail_list
+  retries: 60
+  delay: 60
```

Comment:
Same as above.

Reply:
Added new variable in the latest change.